Carbon/Functions/Get-ApplockerListOfApps.ps1

```diff
@@ -0,0 +1,45 @@
+﻿
+
+function Get-ApplockerListofApps

```

Comment:
Nouns in PowerShell commands should be singular. Also, `Listof`is implied by the `Get` verb, so I would rename this command to `Get-AppLockerApp`. (If you were to keep `Listof`, which you shouldn't, each workd should be capitalized, e.g. `ListOf`.)

Reply:
Also, app locker is cased `AppLocker`.